app/controllers/users_controller.rb

```diff
@@ -160,7 +160,7 @@ def handle_settings_tab
 
     case @tab
     when "organization"
-      @organization = @user.organization || Organization.new
+      handle_organization_tab
```

Comment:
I think `UsersController` starts to grow too much, and it has many responsibilities. It reminds me of the `StoriesController` (which I hope to refactor one day :) Probably, it could be reorganized, e.g. moving out settings or organization settings, but I don't have a specific solution right now.

Reply:
Sorry just seeing this now. Yeah I agree. I think in terms of refactoring it would be out of scope of this PR, and we could refactor `UsersController` as a separate PR. :)

Reply:
I'm happy to tackle the refactoring after this PR is merged. I plan to use submodule, so `Users::SettingsController` with a method for each tab.